keyboards/boardwalk/keymaps/nchristus/keymap.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "boardwalk.h"
```

Comment:
Please change this to:

```suggestion
#include QMK_KEYBOARD_H
```

Reply:
Updated in https://github.com/qmk/qmk_firmware/pull/5195/commits/e23d9758c802d43be8763bd7982cfec5220f1d82